website/docs/r/vna_appliance_v1.html.markdown

```diff
@@ -25,17 +25,22 @@ resource "ecl_vna_appliance_v1" "appliance_1" {
 	availability_zone = "zone1-groupb"
 	virtual_network_appliance_plan_id = "6589b37a-cf82-4918-96fe-255683f78e76"
 
-	interfaces_1_info {
+	interface_1_info {
 		name = "interface_1"
 		network_id = "30f50994-b860-41f1-ba5b-87d9da7fd78a"
 	}
 
-	interfaces_1_fixed_ips {
+	interface_1_fixed_ips {
 		ip_address = "192.168.1.10"
 	}
 }
 ```
 
+You have another choices on how to specify fixed_ips:
+
+* If you don’t specify fixed_ips, an IP address is automatically assigned from your subnets and the address is associated with the OS.
+* If you specify `interface_1_no_fixed_ips = "true"` instead of fixed_ips, an IP address is not assigned.
```

Comment:
```suggestion
* If you specify `interface_<slot_number>_no_fixed_ips = "true"` instead of fixed_ips, an IP address is not assigned.
```
細かいですが、slot1だけの話ではないので汎用的な書き方をしたほうが良いかと思います。

Reply:
了解しました。ここは `interface_[slot number]_no_fixed_ips` という記述に改めます。
提案いただいたのは `<slot_number>` でしたが、同じページ内の見出し「Argument Reference」では `[slot number]` で表現していたため、そちらを用いました。